drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -1018,12 +1018,13 @@ void RasterizerCanvasGLES3::_record_item_commands(const Item *p_item, const Tran
 					state.instance_data_array[r_index].flags = base_flags | (state.instance_data_array[r_index - 1].flags & (FLAGS_DEFAULT_NORMAL_MAP_USED | FLAGS_DEFAULT_SPECULAR_MAP_USED)); //reset on each command for sanity, keep canvastexture binding config
 
 					for (uint32_t j = 0; j < 3; j++) {
+						int offset = j == 0 ? 0 : 1;
 						//second half of triangle
```

Comment:
```suggestion
						// Second triangle of the quad.
```

Reply:
Thanks for the review, updated!

Reply:
Good addition about the vertices being used, makes things clearer / `offset` less cryptic. :slightly_smiling_face: